examples/mxnet_mnist.py

```diff
@@ -1,29 +1,36 @@
-# Step 0: import required packages
 import argparse
 import logging
 import os
 import zipfile
+import time
 
-import horovod.mxnet as hvd
 import mxnet as mx
+import horovod.mxnet as hvd
+from mxnet import autograd, gluon, nd
 from mxnet.test_utils import download
 
 # Training settings
 parser = argparse.ArgumentParser(description='MXNet MNIST Example')
```

Comment:
This may be a general question for all horovod examples: 
There are a lot of examples for ResNet on ImageNet. This is great. But I do not find instructions on how to run those example (e.g. with x learning rate and y batch size on z GPUs this example converges to 75.5% accuracy). This will help people validate that everything is working when they try out the example.

Reply:
@eric-haibin-lin The default values are the ones we used to achieve 75.5% reported. Users can reproduce this using the defalt argument values.